webpack.config.js

```diff
@@ -1,17 +1,28 @@
+require('dotenv').config();
 const path = require('path');
+const glob = require('glob-all');
 const webpack = require('webpack');
+const TerserPlugin = require('terser-webpack-plugin');
+const PurgecssPlugin = require('purgecss-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const root = __dirname.substring(0, __dirname.lastIndexOf('/') + 1);
 
+class TailwindExtractor {
+    static extract(content) {
+        return content.match(/[A-Za-z0-9-_:/]+/g) || [];
+    }
+}
 module.exports = {
-    mode: 'development',
+    mode: process.env.NODE_ENV ? 'development' : 'production',
```

Comment:
I don't thinnk this check is correct. This should be `process.env.NODE_ENV || 'development'`

Reply:
i made this so because i made a custom environment called 'review'.....for our review apps..... and Webpack only has none, development and production environments....... so this code makes the any other environment we have besides development become production so webpack can compile for production in the review app section of heroku

Reply:
i didn't make it staging because there is an app called staging in heroku..... which at any moment could be customized by us for any reason.... in summary i just wanted the naming to be intact for later usage......